### PR TITLE
Make community IDs more easily selectable

### PR DESCRIPTION
**Pull request by TheLastZombie**
A simple patch so that users can more easily subscribe to communities on other platforms.

Requires https://github.com/LemmyNet/lemmy-translations/pull/59.

Before:
firefox_su0m4BnvqE

After:
firefox_Z17P2VMF0z

### CODE REVIEW

File: src/shared/components/community/sidebar.tsx [hunk 1]
```diff
@@ -144,10 +144,10 @@ export class Sidebar extends Component<SidebarProps, SidebarState> {
             {myUSerInfo && this.blockCommunity()}
             {!myUSerInfo && (
               <div className="alert alert-info" role="alert">
-                {i18n.t("community_not_logged_in_alert", {
-                  community: name,
-                  instance: hostname(actor_id),
-                })}
+                {i18n.t("community_not_logged_in_alert")}
```
Comment (SleeplessOne1917) [inline src/shared/components/community/sidebar.tsx:147]: Try using the T component. Here's a code snippet example from the modlog component:

```
<T i18nKey="modlog_content_warning" class="d-inline">
    #<strong>#</strong>#
</T>
```

**Review by SleeplessOne1917 — CHANGES_REQUESTED** [review]
Marking as request changes until my earlier comment is addressed.

**Comment by TheLastZombie** [comment]
Like this?

File: src/shared/components/community/sidebar.tsx [hunk 2]
```diff
@@ -144,10 +145,10 @@ export class Sidebar extends Component<SidebarProps, SidebarState> {
...
             {!myUSerInfo && (
               <div className="alert alert-info" role="alert">
-                {i18n.t("community_not_logged_in_alert", {
-                  community: name,
-                  instance: hostname(actor_id),
-                })}
+                <T i18nKey="community_not_logged_in_alert">#</T>
+                <code className="user-select-all">
+                  !{name}@{hostname(actor_id)}
+                </code>
```
Comment (SleeplessOne1917) [inline src/shared/components/community/sidebar.tsx:151]:
```suggestion
<T i18nKey="community_not_logged_in_alert" interpolation={{community: name, instance: hostname(actor_id)}}>
    #<code>#</code>
</T>
```

You'll need to change your translation in the translations repo to be like this:
`"You are not logged in. However you can subscribe from another Fediverse account, for example Lemmy or Mastodon. To do this, paste the following into the search field of your instance: <1>!{{community}}@{{instance}}</1>"`

**Comment by SleeplessOne1917** [comment]
@TheLastZombie I made a suggested change. Hopefully that will help clear things up.

File: src/shared/components/community/sidebar.tsx (hunk 2 above)
Comment (TheLastZombie) [reply src/shared/components/community/sidebar.tsx:151]: I see, thanks.

**Comment by TheLastZombie** [comment]
This should work. I added another `#` at the end and reverted the French translation.

**Review by SleeplessOne1917 — APPROVED** [review]

**Review by dessalines — APPROVED** [review]
Looks good, thx!

**Pull request merged**